cmd/bootkube/render.go

```diff
@@ -195,3 +224,28 @@ func altNamesFromURLs(urls []*url.URL) *tlsutil.AltNames {
 	}
 	return &an
 }
+
+func pickDNSServiceIP(serviceCIDR string) (string, error) {
+	ip, ipnet, err := net.ParseCIDR(serviceCIDR)
+	if err != nil {
+		return "", err
+	}
+	// pick the 10th IP as a conventional default
+	for i := 0; i < 10; i++ {
+		incIPv4(ip)
```

Comment:
I believe using `ipnet.IP` from `ip, ipnet, err := net.ParseCIDR()` should be sufficient. Maybe just sanity check that it actually returns the first IP in the cidr range even if you're specifying a different address in the string (e.g. `10.1.1.200/24` we should end up with `ipnet.IP=10.1.1.0` and we can increment that value.